compute/os_login_ssh_key/main.tf

```diff
@@ -0,0 +1,7 @@
+data "google_client_openid_userinfo" "me" {
+}
+
+resource "google_os_login_ssh_public_key" "default" {
+  user = data.google_client_openid_userinfo.me.email
+  key  = file("path/to/id_rsa.pub")
```

Comment:
Can we replace with mock data like in https://github.com/terraform-google-modules/terraform-docs-samples/pull/204/commits/e4d691f4678a103d9bfc8ed9b33b7485e8222ef4

Reply:
We might also need to skip int test for this one